README.md

```diff
@@ -163,3 +163,27 @@ defined by a composition of a translation and a linear transformation. An
 (and controllable) if you construct it from a composition of a linear map
 and a translation, e.g. `Translation(v) ∘ LinearMap(v)` (or any combination of
 `LinearMap`, `Translation` and `AffineMap`).
+
+#### Perspective transformations
+
+The perspective transformation maps real-space coordinates to those on a virtual
+"screen" of one lesser dimension. For instance, this process is used to render
+3D scenes to 2D images in computer generated graphics and games. It is an ideal
+model of how a pinhole camera operates and is a good approximation of the modern
+photography process.
+
+The `PerspectiveMap()` command creates a singleton `Transformation` to perform
```

Comment:
Nitpick: can we drop the mention of the fact that this is a singleton? In my opinion the fact that an "empty type" (ie, one without any fields) may be represented as a singleton is an implementation detail of the compiler, and very confusing terminology for people coming from other languages, where singleton means something rather different.